travis/setup_conda.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Environment
+LIBGFORTRANVER="3.0"
+SHERPA_CHANNEL=sherpa
+XSPEC_CHANNEL=cxc/channel/dev
+MINICONDA=$HOME/miniconda
+
+if [[ ${TRAVIS_OS_NAME} == linux ]];
+then
+    MINICONDA_OS=Linux
+    export COMPILERS="gcc_linux-64 gxx_linux-64 gfortran_linux-64"
+    sed -i.orig "s|#extra-fortran-link-flags=|extra-fortran-link-flags=-shared|" setup.cfg
+else  # osx
```

Comment:
I don't think Travis has any plans for supporting Windows, and neither do we. If that changes we'll notice and it will be easy to fix (except that everything else will break on Windows or not even run), so it's safe to assume it's either mac or linux.